Samples/WMCConsole_winrtcpp/WMCConsole_winrtcpp.vcxproj

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
>Project [](start = 1, length = 7)

how is this compiled without the CPPWinRT NuGet package dependency?

Reply:
also, is this using C++/WinRT 2.0+?

---
In reply to: [284522941](https://github.com/microsoft/Windows-Camera/pull/9#discussion_r284522941) [](ancestors = 284522941)

Reply:
This is only using the language extension... and not creating RT classes using IDL. This the package is not required, just a C++17 compiler is enough.

---
In reply to: [284524812](https://github.com/microsoft/Windows-Camera/pull/9#discussion_r284524812) [](ancestors = 284524812,284522941)

Reply:
i guess the version of c++/Winrt matters only to the component generating the classes from idl and might affect if we are trying to generate winrt headers from winmds etc. Here non of the cases are applicable.

---
In reply to: [284524812](https://github.com/microsoft/Windows-Camera/pull/9#discussion_r284524812) [](ancestors = 284524812,284522941)